src/test/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSparkUnitTest.java

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.tools.picard.analysis.InsertSizeMetrics;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public final class CollectInsertSizeMetricsSparkUnitTest extends CommandLineProgramTest{
+
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "picard/analysis/CollectInsertSizeMetrics");
+
+    public String getTestedClassName() {
+        return CollectInsertSizeMetricsSpark.class.getSimpleName();
+    }
+
+    @DataProvider(name="metricsfiles")
+    public Object[][] insertSizeMetricsFiles() {
+        return new Object[][] {
+                // {"insert_size_metrics_test.sam", null},
+                {"insert_size_metrics_test.bam", null}//,
+                //{"insert_size_metrics_test.bam", hg19_chr1_1M_Reference}
```

Comment:
Aren't they the same? The cram unit test file is empty, so essentially it is just another sam file, like the bam file case.